core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -300,6 +300,37 @@ class CommunicationPlan
         MPI_Comm_dup( comm, &_comm );
     }
 
+    /*!
+      \brief Copy constructor.
+    */
+    CommunicationPlan( const CommunicationPlan &cp )
+    {
+        MPI_Comm_dup( cp.comm(), &_comm );
+        _neighbors = cp._neighbors;
+        _total_num_export = cp._total_num_export;
+        _total_num_import = cp._total_num_import;
+        _num_export = cp._num_export;
+        _num_import = cp._num_import;
+        _num_export_element = cp._num_export_element;
+        _export_steering = cp._export_steering;
```

Comment:
We need to discuss that copying the view.  If it was a deep copy, then `CommunicationPlan` would have value semantics.  But with that lien it neither has value nor reference semantics which makes it harder to reason about.